### PR TITLE
Fix dataset split in KWS20 dataset

### PR DESCRIPTION
**Pull request by rotx-maxim**

### CODE REVIEW

File: datasets/kws20.py [hunk 1]
```diff
@@ -416,7 +410,7 @@ def __gen_datasets(self):
             torch.save(mfcc_dataset, os.path.join(self.processed_folder, self.data_file))
 
         print('Dataset created!')
-        print('Training: %d,  Validation: %d, Test: %d' % (traincount, validatecount, testcount))
+        print('Training/Validation: %d,  Test: %d' % (traincount, testcount))
```
Comment (MaximGorkem) [inline datasets/kws20.py:413]: Instead of "Training/Validation" "Training+Validation" can be more understandable. Same applies in the whole document.

**Review by MaximGorkem — DISMISSED** [review]
Good change.

**Review by ermanok — APPROVED** [review]
This is OK. No need to generate the validation set in advance with our current dataset loader.

File: datasets/kws20.py (hunk 1 above)
Comment (rotx-maxim) [reply datasets/kws20.py]: Good idea, will do this ASAP.

**Pull request merged**